rcl/include/rcl/security.h

```diff
@@ -0,0 +1,138 @@
+// Copyright 2018 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef RCL__SECURITY_H_
+#define RCL__SECURITY_H_
+
+#ifdef __cplusplus
+extern "C"
+{
+#endif
+
+#include <stdbool.h>
+
+#include "rcl/allocator.h"
+#include "rcl/types.h"
+#include "rcl/visibility_control.h"
+#include "rmw/security_options.h"
+
+#ifndef ROS_SECURITY_DIRECTORY_OVERRIDE
+# define ROS_SECURITY_DIRECTORY_OVERRIDE "ROS_SECURITY_DIRECTORY_OVERRIDE"
+#endif
+
+#ifndef ROS_SECURITY_ROOT_DIRECTORY_VAR_NAME
+# define ROS_SECURITY_ROOT_DIRECTORY_VAR_NAME "ROS_SECURITY_ROOT_DIRECTORY"
+#endif
+
+#ifndef ROS_SECURITY_LOOKUP_TYPE_VAR_NAME
+# define ROS_SECURITY_LOOKUP_TYPE_VAR_NAME "ROS_SECURITY_LOOKUP_TYPE"
+#endif
+
+#ifndef ROS_SECURITY_STRATEGY_VAR_NAME
+# define ROS_SECURITY_STRATEGY_VAR_NAME "ROS_SECURITY_STRATEGY"
+#endif
+
+#ifndef ROS_SECURITY_ENABLE_VAR_NAME
+# define ROS_SECURITY_ENABLE_VAR_NAME "ROS_SECURITY_ENABLE"
+#endif
+
+/// Init the security options from the values of the environment variables and passed names.
```

Comment:
@ivanpauno nit: consider 
```suggestion
/// Initializes security options from values in the environment and given names.
```

Reply:
I will use imperative mood instead of indicative mood, but otherwise follow your suggestion.

Reply:
Done in https://github.com/ros2/rcl/pull/515/commits/a905ae9dc01d24d4b09f0999f538b61a0e83315e.